spring-cloud-dataflow-rest-resource/src/main/java/org/springframework/cloud/dataflow/rest/resource/AbstractDefinitionAppStatusResource.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2018 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.dataflow.rest.resource;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.springframework.hateoas.ResourceSupport;
+
+/**
+ * Resource representing the status of an app in a definition.
+ *
+ * @author Glenn Renfro
+ */
+public abstract class AbstractDefinitionAppStatusResource extends ResourceSupport {
+
+	private String appName;
+	private String  DSL;
```

Comment:
extra space and should we change to lowercase format throughout the PR as that's how `dsl` is usually used.